controller/user.controller.js

```diff
@@ -0,0 +1,31 @@
+var User = require('../model/user');
+const bcryptjs = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+
+module.exports.register = async(req, res) => {
+    var user = await User.find();
+
+    res.json(user)
+}
+module.exports.Postregister = async(req, res) => {
+
+
+    var hash = bcryptjs.hashSync(req.body.pass);
+    console.log(hash);
+    req.body.pass = hash;
+    var user = await User.create(req.body);
+    res.json(user)
```

Comment:
thêm mã trả về nữa. http response status codes (https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)
```suggestion
    res.status(201).json(user)
```

Reply:
ok pro